cassini_v2_files/mission_phase_name.py

```
@@ -0,0 +1,268 @@
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('directorypath', type=str, metavar='DIRECTORY_PATH',
+                        help='The path to the directory')
+    parser.add_argument('pattern', type=str, metavar='PATTERN',
+                        help='The extension of the labels you want to fix')
+    parser.add_argument('kernel_directory', type=str, metavar='KERNEL_DIRECTORY',
+                        help='The path to the kernel directory')
+
+    args = parser.parse_args()
+
+    with open("mission_data.json", "r") as file:
```

Ruff (0.15.6)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)